### PR TITLE
Update concept mapping for concept list submission preparation

### PR DESCRIPTION
**Pull request by tresoldi**
This PR prepares code and data for later concept list submission to Concepticon.

What I did:

- reviewed mapping by studying entries; some entries were excluded because the reported value included an object (while the mapping was the verb concept), other were extended (multiple glosses with the same mapping) because it was just a glossing detail (and the values are comparable)
- sorted concepts, added a Number column, generating IDs with `id_factory`
- update CLDF files with the newest Glottolog and Concepticon versions

Once this is accepted, I will prepare a Concepticon submission (only this list, no new concept sets, even though some could arguably be included, as IVORY and NOSE RING) -- or let @chrzyki and @Kristina-Pianykh do it, as preferred.

### CODE REVIEW

**Comment by tresoldi** [comment]
Pinging @LinguList as well.

**Comment by chrzyki** [comment]
Hm, the concept part of the PR is a little bit hard to review due to the complex diff/change in ordering. Can have a closer look later from my main computer.

**Comment by chrzyki** [comment]
Oh, and of course thanks for tackling this. :)

**Comment by tresoldi** [comment]
Right, sorry for that. I will commit back in the previous order so you can review, and reverted to the sorted one later.

**Comment by tresoldi** [comment]
@chrzyki done, in original order.

File: etc/concepts.tsv [hunk 1]
```diff
@@ -209,20 +209,20 @@ Quiver	Quiver	995	QUIVER
 Greegree	Greegree	2804	AMULET
 Sacrifice	Sacrifice	1103	SACRIFICE
 Hell	Hell	878	HELL
-Waterfresh	Water (fresh)		
+Waterfresh	Water (fresh)	948	WATER
```
Comment (Kristina-Pianykh) [inline etc/concepts.tsv]: Maybe leave it unmapped? In the Concepticon, WATER refers very broadly to any H₂O liquid, including that from rivers/body tissues, etc.

**Review by Kristina-Pianykh — COMMENTED** [review]
@tresoldi you seem to have a better understanding of the list and its glosses so my comments are not going to be relevant then. The main things:

1. Some mappings are used repetitively (e.g., YOUNG, OLD (AGED), ARMLET, etc.). If it's not a problem for this specific list and it calls for an exception, then ignore this comment.

2. Some glosses seem too specific to me to be mapped to more general things (e.g., Lizard (the common one) (male), Gold (brass), Spoon (of wood), etc.). But then again, if the forms, used for these similar glosses are comparable, then ignore this comment.

**Comment by tresoldi** [comment]
@Kristina-Pianykh I checked your comments because I remember how much you worked on this, the changes I propose are mostly if not all linked to the contents in the data. I might be wrong in favouring the lexical data, though, and I am happy to revert.

But to demonstrate the issue: the case you mention, of "water (fresh)", is used in this dataset for a single lexeme, in Joola, and it has the same value of "water" (`mómẹl`). My guess is that it comes from combining different field works/sources. I revised other glosses, such as `Foot, or Instep of the Foot`, in cases where the new concepticon mapping was missing and it seemed appropriate from research (in the foot case, the words reported are "foot" in wider sense and "inner foot" in a stricter one).

I can see how this might be a problem when reusing the list though, especially for the first line of reasoning (that is, cases like "water (fresh)"). Maybe we could do the following, to stay on the safer side:

- I remove the mapping for "I drink water" (the entries are full translations, with the object in all cases and sometimes the pronouns, so it cannot be mapped to just DRINK)
- Keep the new mapping for "HAND" and "FOOT" (as above)
- Keep the new mapping for ARMLET (I'm confident enough from the morphologies)
- Remove the new mappings for "Water (fresh)", "Lizard (male)", "town (walled)" (this last one is just a transparent Arabic borrowing)
- Remove the new mapping for "Needle (for ornament)" (not 100% sure it is the same word indeed)
- Remove the new mapping for GOLD (it colexifies with BRASS, but we don't have a GOLD OR BRASS in Concepticon)
- Other cases are just different glosses, such the repeated EWE and YAM, and they would be kept.

How does it sound? I'd change the file and confirm with you before merging, of course.

**Comment by Kristina-Pianykh** [comment]
All the above suggestions sound very reasonable to me.

Focusing on the lexical data seems the right approach in this case of ambiguous labeling. The only issue that may arise in the future is reviewing the concept labels because as you see yourself it diverges from the conventional strategy we employ in the conception. I think it happens with the lists, mapped long ago, and then the temptation to remap the glosses without looking into the actual lexical data is real :)

Thank you for such a detailed and structured reply @tresoldi!

**Comment by LinguList** [comment]
thanks, this is very fine by me

**Comment by tresoldi** [comment]
Great, I'll push the new list here (so that the CLDF data is up to date) and prepare the Concepticon PR (so we can discuss in better detail if necessary). Thanks @Kristina-Pianykh and @LinguList

**Pull request merged**